lib/y2_installbase.pm

```diff
@@ -550,7 +550,12 @@ sub save_remote_upload_y2logs {
 sub post_fail_hook {
     my $self = shift;
 
-    if (check_var("REMOTE_CONTROLLER", "ssh") || check_var("REMOTE_CONTROLLER", "vnc")) {
+    if (check_screen 'error_during_installation') {
```

Comment:
it would be more clear something like `linuxrc-installation-error`, but inside of `get_to_console` looks like it is already named as `linuxrc-install-fail`